cmd/get/get.go

```diff
@@ -18,27 +20,121 @@ func CreateGetCommand() *cobra.Command {
 		Short: "Retrieve various resource.",
 	}
 
-	getResourcesFuncs := map[string]core.CommandFunc{
+	getResourcesFuncs := map[string]cmdCore.CommandFunc{
 		"projects": getProjectsFunc,
+		"domains":  getDomainFunc,
+		"tasks":    getTaskFunc,
+		"workflows":    getWorkflowFunc,
 	}
 
-	core.AddCommands(getCmd, getResourcesFuncs)
+	cmdCore.AddCommands(getCmd, getResourcesFuncs)
 
 	return getCmd
 }
 
-func getProjectsFunc(ctx context.Context, args []string, cmdCtx core.CommandContext) error {
+func getProjectsFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
 	projects, err := cmdCtx.AdminClient().ListProjects(ctx, &admin.ProjectListRequest{})
 	if err != nil {
 		return err
 	}
-
 	logger.Debugf(ctx, "Retrieved %v projects", len(projects.Projects))
 	printer := tableprinter.New(cmdCtx.OutputPipe())
 	printer.Print(toPrintableProjects(projects.Projects))
 	return nil
 }
 
+func getTaskFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
```

Comment:
can we split these functions into seprate files, like "get/workflow.go", "get/task.go"

Reply:
ok